Add a Connections Listener to BDK

anthony-symphony opened the pull request:
Add a Connections Listener to BDK
Fix some Windows based tests

symphony-thibault reviewed (approved):
LGTM

symphony-youri commented on symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/main/java/com/symphony/bdk/bot/sdk/event/model/ConnectionAcceptedEvent.java:
```diff
@@ -0,0 +1,24 @@
+/*
```
Please change your default template as to not have those comments :)

symphony-youri commented:
Whats the use case for this listener? (just to get a bit more context and understand why it is useful, could be in the commit message to retrieve it easily by looking at the history)

anthony-symphony commented:
Added reasoning and removed the comments. Reasoning is to allow bots that need to communicate externally to have the ability to listen to incoming connection requests and accept them.
